pkgs/by-name/do/doge/package.nix

```diff
@@ -6,22 +6,18 @@
 }:
 python3Packages.buildPythonPackage rec {
   pname = "doge";
-  version = "3.6.0";
+  version = "3.7.0";
 
   src = fetchFromGitHub {
     owner = "Olivia5k";
     repo = "doge";
     rev = version;
-    hash = "sha256-72nRghD5k0ofrlvV3hEdyrr6uzV4+8At1bOCmRZTxhk=";
+    hash = "sha256-LmEbDQUZe/3lg/Ze+WUNyYfC8zMr88/rn10sL0jgbGA=";
   };
 
-  patches = [
-    # https://github.com/Olivia5k/doge/pull/66, adds a small doge
-    (fetchpatch {
-      url = "https://github.com/Olivia5k/doge/commit/14e3ccc0a3f1e91862492e20a34d008768a34039.patch";
-      hash = "sha256-3F+7F9RuoiVWoN+69T7tM871AXX1IQbHqFxD+S3TjxQ=";
-    })
-  ];
+  pyproject = true;
+  propagatedNativeBuildInputs = [python3Packages.setuptools];
```

Comment:
why is this not `nativeBuildInputs` ?

Reply:
I'm not sure, but it didn't work with nativeBuildInputs, so I thought maybe propagate them, and it worked.

Reply:
nevermind it works with `nativeBuildInputs`